### PR TITLE
App strings reworked, nb_NO added

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Review by premnirmal — CHANGES_REQUESTED** [review]
Thanks for adding the translations. But there are lots of unwanted and unrelated changes, please remove those

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,46 +0,0 @@
-# Contributor Covenant Code of Conduct
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply CODE_OF_CONDUCT.md:1]: I don't want to contribute to authoritarianism. As a token of goodwill I offer my contribution on those terms.

File: app/src/main/res/values-nb-rNO/strings.xml [hunk 2]
```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+
+  <string name="widget_label" translatable="false">Stocks Tracker Widget</string>
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply app/src/main/res/values-nb-rNO/strings.xml:4]: They don't actually do anything.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -34,34 +34,34 @@
...
   <string name="text_color">Text color</string>
-  <string name="start_time">Start time</string>
-  <string name="end_time">End time</string>
+  <string name="start_time">Start-time</string>
+  <string name="end_time">End-time</string>
   <string name="update_interval">Update interval</string>
   <string name="update_days">Update days</string>
   <string name="update_days_desc">Updates will occur on these days</string>
-  <string name="round_two_dp">Round to 2 dp</string>
+  <string name="round_two_dp">Rounded numbers</string>
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:49]: What is the point in just fixing the nb_NO strings without bringing the improvements to the source language?

**Pull request closed without merging**